### PR TITLE
semantic-release

### PR DESCRIPTION
**Pull request by abernier**
"kill all humans"
— Bender

### CODE REVIEW

**Comment by talentlessguy** [comment]
why reset the version?

**Comment by abernier** [comment]
> why reset the version?

Semantic release takes care of updating the version just before publishing to NPM. There is no need to have it versioned. This is also how it's done in drei and explained in more details here: https://semantic-release.gitbook.io/semantic-release/support/faq#why-is-the-package.jsons-version-not-updated-in-my-repository

File: .node-version
Comment (abernier) [inline .node-version:1]: "`.node-version` is a file read by various tools"[^1] among `actions/setup-node`.

Proof it was taken into account in our `ci` pipeline: https://github.com/pmndrs/react-postprocessing/actions/runs/4850486079/jobs/8643428503#step:3:9

[^1]: https://stackoverflow.com/questions/27425852/what-uses-respects-the-node-version-file/62978089#62978089

**Comment by drcmda** [comment]
thanks @abernier !

**Pull request merged**